demo/core_10_base.js

```diff
@@ -418,7 +418,7 @@ $.user.think.iobj = 'any';
 $.user.eval = function(cmd) {
   // Format:  ;1+1    -or-    eval 1+1
   var src = (cmd.cmdstr[0] === ';') ? cmd.cmdstr.substring(1) : cmd.argstr;
-  src = $.utils.code.rewriteForEval(src);
+  src = $.utils.code.rewriteForEval(src, false);
```

Comment:
I think it would be fine to make forceExpression optional (i.e., document as such) and omit an explicit false here.

Alternatively, the style guide now (as of March) notes that you may add a comment to make the purpose of the second parameter more clear: `rewriteForEval(src, /* forceExpression= */ false)`.

Reply:
I quite like the second option.  Done.